frontend/javascripts/oxalis/model_initialization.js

```diff
@@ -433,10 +433,14 @@ function setupLayerForVolumeTracing(
   const fallbackLayerIndex = _.findIndex(layers, layer => layer.name === tracing.fallbackLayer);
   const fallbackLayer = layers[fallbackLayerIndex];
   const boundaries = getBoundaries(dataset);
+  // The frontend does not support uint64 data. Thus if the segmentation is uint64
+  // we treat it as uint32 and save that it originally has the element class uint64.
+  const isUint64Segmentation = tracing.elementClass === "uint64";
```

Comment:
What is `tracing.elementClass` referring to, the `elementClass` or the `originalElementClass`?
The backend converts an `elementClass` of `uint64` to `uint32`, so the `elementClass` value should never be `uint64`.
This is how we achieved the original cheap uint64 support [#4233](https://github.com/scalableminds/webknossos/pull/4233). If you want to, the backend can send the real `elementClass` to the frontend, which would mean we could rid of the `originalElementClass` in the backend.

Reply:
this is in case of volume tracings (as opposed to view mode). the tracing still sends the uint64.
but yeah, feel free to unify this in whichever way seems best :)

Reply:
I think we need to be extra careful before allowing 64 bit volume tracings, as they will probably mix 32 bit and 64 bit data. But this PR mainly deals with the view mode, right?

Reply:
1. I got no mail notification that youri posted this concern 😕.
> But this PR mainly deals with the view mode, right @MichaelBuessemeyer ?

No it doesn't This PR also enables 64-bit volume tracings in the frontend. If we do not what this behaviour (and that's what I think is your opinion) we just have to remove the changes in lines `436-438, 442`. Then the user will get a warning upon opening a volume tracing. 


@youri: When the user creates a volume tracing the backend creates an empty tracing (maybe including a fallback layer). At this point the backend seems inconsistent, because trying to create a volume tracing on a dataset with a uint64-bit segmentation layer will set the `elementClass` of the initial tracing to `elementClass`. At least that's what I remember and why I needed to add the lines `436-438, 442`.


According to the suggestion to always send the 64-bit element class as the property `elementClass` and getting rid of `originalElementClass`: This should be possible :).

Reply:
I see, thank you for your reply!

We should extensively test the 64 bit volume tracing, especially with how fallback layer data is incorporated into the tracing, and if this works with https://github.com/scalableminds/webknossos/pull/4602
But yeah, let’s try it :)

Reply:
I just had a conversation with Youri and we decided it would be preferrable **not** to allow 64-bit volume tracings yet. 

While the trouble with their data being in 32 bit and the fallback buckets being 64 bit is invisible in the wK frontend, since the fallback buckets will be truncated to 32 bit, the data will not *actually* be compatible, it will not actually be clear which ID should belong to which cell.

Additionally, saving 32 bit data labeled as 64 bit data can lead to incompatibilities in case the frontend in some future version using some future javascript features actually produces 64 bit data. Then our implicit knowledge becomes ambiguous.

Since no user has yet wished for volume tracings with 64 bit fallback layer, and this PR was originally only about the user notification about truncated data, please remove the changes you mentioned, @MichaelBuessemeyer , so that the user can not trace on 64 bit volume tracings. Additionally, @youri-k , please add an check in the back-end side *creation* of volume tracings, so that 64 bit volume tracings can not be created.